xcode/Subconscious/Shared/Components/Detail/UserProfileDetailView.swift

```diff
@@ -144,20 +141,30 @@ enum UserCategory: Equatable, Codable, Hashable, CaseIterable {
 struct UserProfile: Equatable, Codable, Hashable {
     let did: Did
     let nickname: Petname.Name?
-    let address: Slashlink
+    var address: Slashlink
```

Comment:
We need to be able to change this field to achieve https://github.com/subconsciousnetwork/subconscious/issues/613